ADA/ada.qmd

```diff
@@ -36,14 +35,65 @@ The ambition is to enable access to all datasets across DfE and ESFA and retire
 
 ---
 
+# What is Databricks?
+
+Databricks is a browser-based platform developed by Microsoft which allows you to use notebooks to access and analyse data. You can also use it for machine learning. Databricks notebooks support multiple programming languages, including R, Python, and SQL. Unlike RStudio or SQL Server Management Studio (SSMS), you don't have to download any software to be able to use it. You will need to be given access to Databricks and the data you require by the ADA team by completing an "Analytical Data Access Request" service request form in the IT Help Centre. At the moment, not all of the data currently available in SSMS has been migrated to the Delta Lake for use in Databricks. You can check the status of various datasets [in the data catalogue dashboard](https://app.powerbi.com/groups/me/reports/966389db-2e4e-42b6-8416-36aee2fb9503/ReportSection?ctid=fad277c9-c60a-4da1-b5f3-b3b8b34a82f9&experience=power-bi) 
+
+Accessing data through Databricks works slightly differently to the way that you might currently access your data in SSMS. You can read more about how Databricks works, including data storage and how to use clusters, on the [ADA guidance site](https://analytical-data-access.azurewebsites.net/get-support/how-databricks-works). 
+
+Notebooks are made up of multiple code or markdown cells. They are different to IDEs (Integrated Development Environments) like RStudio. You can read more about the differences between IDEs and notebooks and where each might be useful [on the NHS-R website](/ADA/ada.html#differences-between-ides-and-notebooks)
+
+One benefit of Databricks is the use of workflows to run your SQL code. Workflows allow you to automate the running of single or multiple SQL scripts, so if you have large amounts of data processing to do on a regular basis, you can set up a workflow to complete this overnight. Your laptop does not need to be switched on for this to work. 
+
+---
+
+# How will I know when I need to move to Databricks?
+
+At the moment, migration to Databricks is voluntary. If your team has been onboarded then you should be contacted by the ADA team with more information, and you should be offered onboarding training to learn more about the platform and how to use it. 
+
+Once your team's data has been migrated, you will **not** lose access to your existing data sources, so you will be able to dual run your code using data from SSMS and data from the Databricks SQL warehouse to ensure that everything is working as it should be. 
+
+---
+
+# What will I have to change in my existing code? Do I need to stop using RStudio and move everything to Databricks? 
```

Comment:
Following my earlier comment where I suggest these

to take advantage of cloud computing may need to translate existing code to languages supported within databricks. (can link to more info below and relevant ADA guidance)

and 

For teams that have invested time into RAP and have existing code pipelines that work well for them it is possible to make use of data in ADA and cloud computing via databricks without having to make significant changes (link to detail below).

I think it would make sense to split this section into two - one making the most of databricks without investing to make changes and one how to make the most of databricks offer. 

For the latter it should be framed around issues teams might see and where databricks can solve it for them - not a for the sake of it kind thing. 

So do your scripts take years to run ... databricks lets you do x

Do you run the same query every week ... databricks lets you do y etc